max.go

```
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"unicode/utf8"
+)
+
+func max(x []string) string {
+	var sliceLen, p int
+	if len(x) <= 0 {
+		fmt.Print("Slice has no elements")
```

It's better to return an error or panic